src/layouts/Layout.jsx

```diff
@@ -51,11 +86,16 @@ class Layout extends React.PureComponent {
                           'freeCodeCamp/assets/master/assets/' +
                           'logos/FCC-logo-white.png'
                           }
+                          style={style.Image}
                       />
                     </Link>
                   </Navbar.Brand>
                 </Navbar.Header>
               </Col>
+              <Col md={ 9 } xs={ 6 }>
+                <FCCSearchBar
```

Comment:
We are not using this module for the time being. Instead we are using `SearchBar` from `./components/search/SearchBar.jsx` 

This will resolve as an error